pom.xml

```diff
@@ -43,6 +43,25 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-test</artifactId>
         </dependency>
+
+        <!-- PDF -->
+        <dependency>
+            <groupId>org.apache.pdfbox</groupId>
+            <artifactId>pdfbox</artifactId>
+            <version>3.0.3</version>
+        </dependency>
+        <dependency>
+            <groupId>org.xhtmlrenderer</groupId>
+            <artifactId>flying-saucer-pdf</artifactId>
+            <version>9.9.4</version>
+        </dependency>
+
+        <!-- DOCX / XLSX -->
+        <dependency>
+            <groupId>org.apache.poi</groupId>
+            <artifactId>poi-ooxml</artifactId>
+            <version>5.3.0</version>
+        </dependency>
```

Comment:
Do we really want to add those dependencies to the project? We might not have PDF and Docx / Xlsx in all our projects.

@alexlanz @Piiit Wdyt?

Reply:
I personally would not add it.

Reply:
It only adds those dependencies to the `test` code because this library gets imported with scope test. Flying Saucer is unnecessary indeed. I will remove it.

Reply:
PDF is quite common, maybe that is used in most projects, and could therefore be added, would not add the other two... nevertheless, we could have separate libs to handle such files, and those libs could then have testing tools.

Reply:
Ok, I'll consider this PR to be rejected and close it then.

Reply:
Not sure about that, would add the PDF assert in the meantime, until we have a dedicated lib... wdyt?